integration/solana/minbalance.sol

```diff
@@ -0,0 +1,16 @@
+import '../../solana-library/minimum_balance.sol';
+
+contract minbalance {
+    /// Anchor requires view functions to have a return value
+    function test1() pure public returns (bool) {
```

Comment:
Isn't this a pure function?

Reply:
Anchor does not distinguish between `pure` and `view`.

Reply:
I've improved the comment